changelog/README.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 28th March 2025
+* [Get all service order notes](../operations/serviceordernotes.md#get-all-service-order-notes):
+  * Extended request object with `Types` filtering parameter and support for `SpecialRequest` service order note type.
+* [Get all reservations (ver 2023-06-06)](../operations/reservations.md#get-all-reservations-ver-2023-06-06):
+  * Extended request object with `ChannelNumbers` filtering parameter.
+* [Get all rates](../operations/rates.md#get-all-rates):
+  * Extended [Rate](../operations/rates.md#rate) response object with `TaxExemptionReason` and `TaxExemptionLegalReference` properties.
+* [Set rates](../operations/rates.md#set-rates) (restricted operation):
+  * Removed `AccountingCategoryId`, `BusinessSegmentId`, `ShortNames`, and `ExternalNames` from [Set rate parameters](../operations/rates.md#set-rate-parameters) request object.
+* [Get all products](../operations/products.md#get-all-products):
+  * Extended [Product](../operations/products.md#product) response object with `TaxExemptionReason` and `TaxExemptionLegalReference` properties.
+* [Get all order items](../operations/orderitems.md#get-all-order-items):
+  * Extended [Order item](../operations/orderitems.md#order-item) response object with `TaxExemptionReason` and `TaxExemptionLegalReference` properties.
+  * Extended [Order item type](../operations/orderitems.md#order-item-type) with `AllowanceDiscount`, `AllowanceBreakage`, and `AllowanceContraBreakage`.
+* [Cancel order items](../operations/orderitems.md#cancel-order-items):
+  * New operation added (restricted).
+* [Get all bills](../operations/bills.md#get-all-bills):
+  * Extended [Bill company data](../operations/bills.md#bill-company-data) response object with the following properties: `DUNS`, `Telephone`, `TaxIdentifier`, `InvoicingEmail`, and `Department`.
+* [Get all availability adjustments](../operations/availabilityadjustments.md#get-all-availability-adjustments):

```

Comment:
As far as I can see, the `FirstTimeUnitReleaseUtc` was already present in the response object, it's just moved location in the docs. So the only change here really is to improve the JSON examples.